frontend/src/app/components/acceleration-timeline/acceleration-timeline.component.ts

```diff
@@ -19,15 +19,22 @@ export class AccelerationTimelineComponent implements OnInit, OnChanges {
   acceleratedAt: number;
   now: number;
   accelerateRatio: number;
+  useAbsoluteTime: boolean = false;
 
   constructor() {}
 
   ngOnInit(): void {
     this.acceleratedAt = this.tx.acceleratedAt ?? new Date().getTime() / 1000;
+    this.now = Math.floor(new Date().getTime() / 1000);
+    this.useAbsoluteTime = this.tx.status.block_time < this.now - 7 * 24 * 3600;
+
+    window.setInterval(() => {
```

Comment:
This timeout continues after user leaves the view.
Set it to a variable and run `clearTimeout(this.timeout)`